cookiecutter.json

```diff
@@ -0,0 +1,14 @@
+{
+    "full_name": "Your Name",
+    "email": "you@yourdomain.com",
+    "github_username": "guardicode",
+    "github_repo": "infection-monkey-template",
+    "default_branch": "main",
+    "project_name": "infection-monkey-template",
+    "package_name": "infection-monkey-template",
+    "project_short_description": "Sample cookiecutter template for infection monkey.",
+    "python_major_version": 3,
+    "python_minor_version": 11,
+    "license": ["MIT", "Apache-2.0"],
```

Comment:
Most of monkey is licensed under GPLv3, so we should add it to this list.

Reply:
I added all 3 license files.